contributions/opening-and-assessing-issues.md

```diff
@@ -109,6 +109,12 @@ GitHub [milestones](https://docs.github.com/en/issues/using-labels-and-milestone
 
 Before a new milestone can be created in the repository, the new milestone must be approved by the project Maintainers, following the decision making rules in the [GOVERNANCE.md](../GOVERNANCE.md) file.
 
-### GitHub project assignments
+## GitHub project assignments
 
 In place of assigning a milestone, consider assigning issues to a GitHub [project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects). An issue can have multiple project assignments, and each Project Maintainer can manage their own GitHub projects and the priorities within those projects.
+
+## Critical priority issues
+
+Such a priority must be used sparingly and only applies to released package issues that severely impact a large number of users. For example, an issue that results in data loss for all users can be considered critical, while an issue causing compile errors for some developers does not warrant a critical priority.
```

Comment:
Strongly disagree that compilation errors for _any_ developer should be considered "not [...] critical". Intentionally locking out any portion of the developer base without a patch release is not a stance we should take.

Reply:
We discussed at the maintainer sync. Every two weeks is the normal cadence of updates.  So other high priority issue can be discussed during these normal sync meetings.

Critical issues are meant to be addressed outside those two weeks.   I'll add some text to clarify this.

Reply:
Ah, good to know that there was missing context. Thanks